batch2/create-batch-tables.sql

```diff
@@ -33,6 +33,8 @@ CREATE TABLE IF NOT EXISTS `batches` (
   `n_failed` INT NOT NULL DEFAULT 0,
   `n_cancelled` INT NOT NULL DEFAULT 0,
   `time_created` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  `time_completed` TIMESTAMP,
+  `cost` DOUBLE NOT NULL DEFAULT 0,
```

Comment:
To avoid issues with rounding, I wonder if the cost should be a BIGINT measured in, say, millicore-milliseconds.  That's around 5B core-hrs, or a $50M cost.

Reply:
I was worried about that, but wanted to get the basic thing right first without having an additional units conversion to worry about it. But I'll add the change now.